nuxeo-features/nuxeo-platform-oauth/src/main/java/org/nuxeo/ecm/platform/oauth2/clients/OAuth2Client.java

```diff
@@ -35,6 +39,8 @@
 @XObject("client")
 public class OAuth2Client {
 
+    protected static final Log log = LogFactory.getLog(OAuth2Client.class);
```

Comment:
`log` is the exception and should be `private` as we don't want reuse from subclasses.